apps/src/code-studio/components/aiTutor/studentAccessToggle.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+import style from './chat-messages-table.module.scss';
+import Toggle from '@cdo/apps/componentLibrary/toggle/Toggle';
+import {StudentAccessData} from '@cdo/apps/aiTutor/types';
+
+interface StudentAccesToggleProps {
+  student: StudentAccessData;
+}
+
+const StudentAccessToggle: React.FunctionComponent<StudentAccesToggleProps> = ({
+  student,
+}) => {
+  const [hasAITutorAccess, setHasAITutorAccess] = useState(
+    !student.aiTutorAccessDenied
+  );
+  const handleToggle = () => {
+    setHasAITutorAccess(!hasAITutorAccess);
+    console.log('hasAITutorAccess', hasAITutorAccess);
```

Comment:
- [ ] delete console.log 
- [ ] implement code to actually update `ai_tutor_access_denied` field of user property 😁